instrumentation/http/bnd.bnd

```diff
@@ -0,0 +1,5 @@
+Import-Package: \
+  !brave.internal*,\
```

Comment:
Why is the exclusion of brave.internal necessary here? Does http access the internal package?

Reply:
well it uses brave.internal.Nullable which is runtime scope
annotation. That's it.. perhaps I'm being overly defensive?